DNN Platform/Library/packages.config

```diff
@@ -11,7 +11,7 @@
   <package id="MimeKit" version="2.10.1" targetFramework="net472" />
   <package id="Newtonsoft.Json" version="10.0.3" targetFramework="net45" />
   <package id="PetaPoco.Compiled" version="6.0.415" targetFramework="net472" />
-  <package id="Portable.BouncyCastle" version="1.8.9" targetFramework="net472" />
+  <package id="Portable.BouncyCastle" version="1.8.8" targetFramework="net472" />

```

Comment:
Any reason for lowering the version or is that a mistake ?

Reply:
The version that MailKit and MimeKit were depending on was 1.8.8. With 1.8.9 in there .NET Kept throwing errors trying to load the dll. Wasnt until I went to 1.8.8 that the loading errors stopped.

Reply:
Interesting... Ok, makes sense then...